### PR TITLE
Update documentations for DG, UG and PPP

### PR DESCRIPTION
**Pull request by AlyssaPng**
Update DG, UG and PPP

### CODE REVIEW

**Review by Kb-Tay — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -448,6 +455,8 @@ UNOFAS data are saved automatically as a JSON file `[JAR file location]/data/add
 
 <div markdown="span" class="alert alert-warning"> :exclamation: **Caution:**
 If your changes to the data file makes its format invalid, UNOFAS will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.
+
+Changing the `appointment` date to an invalid month (e.g. 13-13-2023) will cause an error and UNOFAS will not run. However, changing it to an invalid day (e.g. 31-02-2023), UNOFAS will update this date to the closest correct date (e.g. 28-02-2023).
```
Comment (sopa301) [inline docs/UserGuide.md:459]: We don't need to state this explicitly, we can just say that only correct edits will work, and that any wrong edits will lead to undefined behaviour, including the deletion of the entire data file.
image

```suggestion
If your changes to the data file makes its format invalid, UNOFAS will discard all data and start with an empty data file at the next run. Certain edits to the file may also cause unexpected behaviours. Please only edit the file if you are confident that it is correct, and it is recommended to take a backup of the file before editing it.
```

**Review by sopa301 — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (AlyssaPng) [reply docs/UserGuide.md:459]: okayy will edit

**Pull request merged**